### DISCUSSION TITLE
How to use parameters mode that was introduced in PR #1404

### DISCUSSION THREAD

**Discussion by Gwen2** [allure-framework/allure2#1659] [Questions & Support] [answered]
I found a new feature that it's possible to hide/mask method parameters in Allure report. It was introduced in https://github.com/allure-framework/allure2/pull/1404
It's a great option but I can't figure out how to use it. Can't find description neither in this PR nor in https://docs.qameta.io/allure/

Please point me to the proper place where it's described.

**Comment by baev** [comment, marked as answer]
so far it's only supported for allure-java. See https://github.com/allure-framework/allure-java/pull/648 for details
